ykrt/src/deopt.rs

```diff
@@ -302,7 +375,30 @@ unsafe extern "C" fn __ykrt_deopt(
     // pass in variables from this scope via a struct which is passed into the function.
     LLVMOrcThreadSafeModuleWithModuleDo(moduleref, ts_reconstruct, infoptr as *mut c_void);
 
+    // We want to start side tracing only after we deoptimised so we don't trace the whole deopt
```

Comment:
Perhaps split this sentence in two, at least using a comma?

Reply:
Fixed in 24d4962b8

Reply:
Fixed in cb78bfe39